client/src/components/RatingsComponents/ReviewsList/HelpfulTile.jsx

```diff
@@ -32,11 +35,33 @@ function HelpfulTile ({helpfulnessCount, reviewId, setReviewsCount}) {
         'Authorization': token
       }
     })
-    .then(() => console.log('you have reported review # ', reviewId))
-    .catch((err) => console.log('error reporting review', err));
+      .then(() => console.log('you have reported review # ', reviewId))
+      .catch((err) => console.log('error reporting review', err));
     setReviewsCount(prevState => prevState -1);
   };
 
+  // this one not working
+  // const handleReported = () => {
+  //   Axios.put(`http://127.0.0.1:3000/reviews/${reviewId}/report`, {
+  //     params: {
+  //       review_id: reviewId
+  //     }
+  //   })
+  //     .catch((err) => console.error('error putting report::::', err));
+  //   setReviewsCount(prevState => prevState - 1);
+  // };
+  // const handleReported = () => {
+  //   Axios({
+  //     method: 'put',
+  //     url: `http://127.0.0.1:3000/reviews/${reviewId}/report`,
+  //     params: {
+  //       review_id: reviewId
+  //     }
+  //   })
+  //     .catch((err) => console.error('error putting report::::', err));
+  //   setReviewsCount(prevState => prevState - 1);
+  // };
+
```

Comment:
should we remove this, or will you still need it?

Reply:
leaving for refactoring, idea is that all the requests be routed through server.js eventually